cmd/admin-rpc-client.go

```diff
@@ -462,6 +474,26 @@ func writeTmpConfigPeers(peers adminPeers, tmpFileName string, configBytes []byt
 	return errs
 }
 
+// Backup config contents onto config.json.backup on all nodes.
+func backupConfigPeers(peers adminPeers) []error {
+	// For a single-node minio server setup.
+	if !globalIsDistXL {
+		return []error{peers[0].cmdRunner.BackupConfig()}
+	}
+	errs := make([]error, len(peers))
+	wg := sync.WaitGroup{}
+	for i, peer := range peers {
+		wg.Add(1)
+		go func(idx int, peer adminPeer) {
+			defer wg.Done()
+			errs[idx] = peer.cmdRunner.BackupConfig()
+		}(i, peer)
+	}
+	wg.Wait()
+	// Return errors (if any) received during rename.
```

Comment:
We do not rename. We simply backup. What is the point of `Backup` if we do not roll back in case of WriteQuorum failure in `Commitconfig`